src/components/SideBar/SideBar.tsx

```diff
@@ -52,10 +52,10 @@ export default function SideBar({ children }: { children?: Function }) {
 		{ children: <MenuItem onClick={openWorkspaceComponent}>Add Workspace</MenuItem> },
 	];
 
-	if (!data?.organizationList) return <SidebarSkeleton></SidebarSkeleton>;
+	if (!dashboardData?.organization) return <SidebarSkeleton></SidebarSkeleton>;
```

Comment:
Shouldn't it be during loading only?
For no organization there should be a different empty state

Reply:
When a user signup, an organization is directly attached created to it. This mean atleast 1 organization will always be attached to it